### ISSUE TITLE
Improved handling of legacy enhanced multiframe instances

### ISSUE THREAD

**Issue by fedorov** [OHIF/Viewers#2193]
@afshinmessiah is working on creating legacy enhanced multiframe representation for IDC-hosted MR, PET and CT images.

The resulting converted MF instances are in this store: https://idc-sandbox-000.firebaseapp.com/projects/idc-tcia/locations/us/datasets/afshin_results_00_idc_tcia_mvp_wave0/dicomStores/MULTIFRAME (example here: https://idc-sandbox-000.firebaseapp.com/projects/idc-tcia/locations/us/datasets/afshin_results_00_idc_tcia_mvp_wave0/dicomStores/MULTIFRAME/study/1.3.6.1.4.1.14519.5.2.1.1706.4009.243406849348906151042089406192?seriesInstanceUID=1.3.6.1.4.1.43046.3.1.9.5060456482850075494393251866013736567025)

I will keep here a laundry list of issues identified while testing those MF instances.

- [ ] MRP is not enabled
- [ ] window center/width are not used (they are in `SharedFunctionalGroupsSequence`)
- [ ] tag browser does not show all of the attributes (it looks like per-frame FG is missing, see example above)

**Comment by pieper** [comment]
@swederik @JamesAPetts this will be a good dataset to use for testing the pet/ct offscreen rendering too. 

@fedorov can you add a link to Afshin's converter code for these?

We might want to consider a multiframe-aware tag browser that could collapse the per-frame sequences into an easier to read vector form.  Also now we really need that scrollbar in the tag browser dialog.

**Comment by fedorov** [comment]
@pieper the code is not yet in state intended for reuse. It is not a single converter, most of the work is actually about fixing the input single-frame data, because otherwise the MF cannot be valid. The code to patch SF input is based on Pixelmed, but in python, here: https://github.com/afshinmessiah/PyDicomVerify. The MF conversion is in this PR: https://github.com/MGHComputationalPathology/highdicom/pull/34. I am not sure if either one is up to date with what Afshin has been using. Our main priority now is to apply it to the TCIA subset in IDC, confirm it is working as intended, and only after the functionality is finalized clean up the patcher and finish the highdicom PR. The plan is for the MF writing functionality to eventually be in highdicom.

**Comment by pieper** [comment]
@fedorov my main question was whether these are considered "gold standard" or at least valid multiframes of the type IDC will be producing, or are we debugging both the converter and ohif at the same time.

**Comment by fedorov** [comment]
Yes, we are definitely debugging converter at the same time. But I am not sure at this point we really need a "gold standard" for the specific issues identified in the original issue. Definitely it is not a priority at this point, and is not marked as such.

**Comment by pieper** [comment]
@fedorov see if there are multiframes on TCIA and if so see if the issues are the same and this should be a priority

**Comment by fedorov** [comment]
Surprisingly, I am not seeing any enhanced multiframe images in TCIA. Let's revisit this after Bill C completed ingestion of the current set of data from TCIA.

**Comment by sedghi** [comment]
@igoroctaviano Do you know what is the status on this on latest master?

**Comment by igoroctaviano** [comment]
@fedorov whats needed for this one?

**Comment by fedorov** [comment]
It does not appear that there are any enhanced CT/MR in IDC, and the project that started this discussion is no longer active, so debugging this is not a priority or even candidate from the IDC side at this time.
